specify/specify7
# Avoid throwing errors on workbench validation for undefined treedefid

Fixes #5313

Fix the issue of workbench upload validation by avoiding error throwing due to the treedefid not being specified.

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

1. Create a workbench upload mapping to multiple treedefitems.
2. Check that the validation runs successfully

Here is a specific DB to try out (ojsmnh_2024_10_03):
1. Go to https://ojsmnh20241003wb-production.test.specifysystems.org/specify/workbench/254/ (`v7.9.7` or `production`), logged in as `spadmin` in the Ichthyology collection
3. Validate the data set
4. See the following error:
	```
	{
	  "uploaderstatus": {
	    "operation": "validating",
	    "taskid": "26f94f3b-f6e8-42a8-812e-1ba8e7e2775e"
	  },
	  "taskstatus": "FAILURE",
	  "taskinfo": "ValueError('Multiple treedefitems found for rank Continent')"
	}
	```
5. Go to https://ojsmnh20241003wb-v7962.test.specifysystems.org/specify/workbench/254/ (`v7.9.6.2`), logged in as `spadmin` in the Ichthyology collection
6. Validate the same data set
7. See no error
